### ISSUE TITLE
[New feature request] Disable print complete notification

### ISSUE THREAD

**Issue by dimateos** [manuth/MarkdownConverter#208]
I would love a silent mode. I basically have a pdf viewer opened that refreshes the file changed from disk, so the notification is reduntant and a bit annoying (I have to manually close it?).

Anyway thanks for the great extension! It's great to be able to use the system parser and get the same output as the preview :)

**Comment by neurolag** [comment]
Oh this sounds very easy to implement 😄 
Personally, I guess it might be a good idea to have an option such as `markdownConverter.Notifications` which can be set to `false`, `true` (default) or a set of message IDs to show. I imagine it like this:

Examples:

```js
{
    "markdownConverter.notifications": true
}
```

```js
{
    "markdownConverter.notifications": false
}
```

```js
{
    "markdownConverter.notifications": {
        "default": false,
        "conversionFinished": true
    }
}
```

```js
{
    "markdownConverter.notifications": {
        "default": true,
        "sslError": false
    }
}
```

What are your thoughts?

**Comment by dimateos** [comment]
Yeah absolutely! I was only thinking about disabling the conversion finished one. But seems good to add granularity to be able to keep notifications for error logs. So my guess would be either:
1. Keep it simple and only disable notifications for `conversionFinished` while keeping the errors. Do not allow the user to remove them. This good be good enough for me.
2. As you suggested with your 2 last examples: option per type of notification. Allow the user to remove any/all. This is also easier to extend to more features that you could add in the future.

Cheers!
